packages/react-button/src/components/ToggleButton/ToggleButton.test.tsx

```diff
@@ -1,6 +1,8 @@
 import * as path from 'path';
 import { isConformant } from '@fluentui/react-conformance';
 import { ToggleButton } from './ToggleButton';
+import { checkedBehaviorDefinition } from './checkedBehaviorDefinition';
```

Comment:
Let's revert the changes here too, not relevant anymore

Reply:
changes reverted